pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -605,7 +605,7 @@ func requestConfigByURL(getconfigCtx *getconfigContext, url string,
 				config, ts, err := readSavedProtoMessageConfig(
 					zedcloudCtx, url,
 					ctx.globalConfig.GlobalValueInt(types.StaleConfigTime),
-					checkpointDirname+"/lastconfig", false)
+					checkpointDirname+"/lastconfig", true)
```

Comment:
Does not this make the option `timer.use.config.checkpoint` obsolete? Or is it still being used in other cases?
I wonder if we should get rid of it completely. It seems to be quite error prone in the offline use-cases. And I do not really see what does it help to prevent. Isn't applying any config better than no config, even if it is some time behind the latest (and currently inaccessible) config?

Reply:
It is used in case of read-radio-config for example and in some other cases. IMO this option is dangerous and does not make any sense, because there should be always the "last" state, regardless how outdated it is. BTW, there are some other complains that VM can show up *hours* later after the node is up, this bug can explain this as well.